kipoiseq/extractors/gtf.py

```diff
@@ -343,15 +344,28 @@ def get_utr_from_gtf(
         :param on_error_warn: Do not break on error; instead throw warning.
         """
         if infer_from_cds:
-            cds = CDSFetcher.get_cds_from_gtf(df=df, on_error_warn=on_error_warn)
+            # get start and end of cds for each transcript
+            cds = CDSFetcher.get_cds_from_gtf(df=df, on_error_warn=on_error_warn) \
+                .groupby('transcript_id') \
+                .agg({'Start': min, 'End': max})
+
+            # join cds start and end to utr df
+            utr_df = df.query("Feature == 'transcript'") \
+                .set_index('transcript_id') \
```

Comment:
I set the index earlier because I need it to join the two dataframes, that's why I deleted it there

Reply:
Got it

Reply:
yeah but then we get in trouble with infer_from_cds = False